rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,497 @@
+DCP-extrinsic developers, and it requires generalization of the fewest
+DCP components. Implementation of support for these assays is likely to
+generalize to additional areas of interest to the HCA community such as
+multiplexed antibody imaging and thick-section volumetric imaging for
+both RNA and protein.
+
+The remainder of this document provides additional details on the
+characteristics of image-based transcriptomics assays, why they are the
+right assay class to prototype imaging support in the DCP, their current
+scientific state, the standardization of imaging file formats, and a
+longer-term view of support for single-cell imaging assays. In brief:
+
+  - Support for image-based transcriptomics can be added without need
+    to construct any additional DCP services, and should not require
+    significant generalization of existing ones.
+
+  - Image-based transcriptomics is the right class of assay to
+    initiate imaging support in the DCP because of their similarity to
+    existing DCP sequencing assays and the abundance of labs using
+    single-molecule FISH assays to clarify spatial distributions of
+    cell types identified in RNA-seq experiments.
+
+  - If desired, *all* key validation, specification, and analysis
+    deliverables can be provided by *starfish* and OME engineers,
+    minimizing load on DCP engineering teams.
+
+  - A draft metadata schema has already been built by EBI via
+    collaboration with the SpaceTx consortium, who have delivered
+    image-based transcriptomics datasets which the ingestion service
+    is beginning to test.
+
+  - Based on [<span class="underline">delivery timelines for
+    prerequisite
+    work</span>](https://docs.google.com/document/d/1RQcj450v0WHGUvSm7YwwFQ7sRFoPAOCjRbo63UzyQd0/edit?ts=5c40d3f1#heading=h.4qbf4q96id16),
+    implementation of support for image-based transcriptomics could
+    begin as early as Q2-Q3 2019.
+
+  - While image-based transcriptomics data can be ingested and stored
+    without a visualization capability, significant user demand can be
+    expected similar to that for count matrices.
+
+### **User Stories**
+
+  - As a submitter, I want to upload directories of files representing
+    an imaging experiment for storage in the DCP.
+
+  - As a user with a keyboard, I want to identify images with a given
+    gene for download.
+
+  - As a user with a keyboard, I want to see spots overlayed on an
+    image for QC.
+
+  - As a user with a keyboard, I want to see cells overlayed on an
+    image for QC.
+
+  - As a user with a keyboard, I want to generate a gene x cell matrix
+    for biological analysis of my data and combination with sequencing
+    assays.
+
+## **Scientific "guardrails"**
+
+Image-based transcriptomics assays are still in active development, and
+benchmarking and characterization of these data types is in process. As
+such we expect to utilize community feedback to iterate on scientific
```

Comment:
Who is the "we" here? Does that mean Ambrose and Josh?

Reply:
Thanks, @mckinsel. @ambrosejcarr slipped one past me in a late revision! I'll let him disambiguate.